packages/core-flows/src/definition/cart/steps/get-item-tax-lines.ts

```diff
@@ -9,21 +9,36 @@ import {
   TaxableItemDTO,
   TaxableShippingDTO,
 } from "@medusajs/types"
+import { MedusaError } from "@medusajs/utils"
 import { StepResponse, createStep } from "@medusajs/workflows-sdk"
 import { ModuleRegistrationName } from "../../../../../modules-sdk/dist"
 
 interface StepInput {
   cart: CartWorkflowDTO
   items: CartLineItemDTO[]
   shipping_methods: CartShippingMethodDTO[]
+  force_tax_calculation?: boolean
 }
 
 function normalizeTaxModuleContext(
-  cart: CartWorkflowDTO
+  cart: CartWorkflowDTO,
+  forceTaxCalculation: boolean
 ): TaxCalculationContext | null {
   const address = cart.shipping_address
+  const shouldCalculateTax = forceTaxCalculation || cart.region?.automatic_taxes
+
+  if (!shouldCalculateTax) {
+    return null
+  }
+
+  if (!address || !address?.country_code) {
+    if (forceTaxCalculation) {
```

Comment:
```suggestion
  if (forceTaxCalculation && !address?.country_code) {
```

Reply:
@carlos-r-l-rodrigues we want this to skip if there is no country code and only throw if its forcing a calculation

Reply:
doing this breaks the flow of `forceTaxCalculation=false` to return with null if address.country_code is not present. We could remove the `!address` or break it into 2 if conditions

This is the expected flow:
```
if (forceTaxCalculation && !countryCode) {
  throw error
}

if (!countryCode) {
  return null
}
```